src/pkg/backup/details/chats.go

```diff
@@ -0,0 +1,120 @@
+package details
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/alcionai/clues"
+
+	"github.com/alcionai/corso/src/pkg/dttm"
+	"github.com/alcionai/corso/src/pkg/path"
+)
+
+// NewChatsLocationIDer builds a LocationIDer for the chats.
+func NewChatsLocationIDer(
+	category path.CategoryType,
+	escapedFolders ...string,
+) (uniqueLoc, error) {
+	if err := path.ValidateServiceAndCategory(path.TeamsChatsService, category); err != nil {
+		return uniqueLoc{}, clues.Wrap(err, "making chats LocationIDer")
+	}
+
+	pb := path.Builder{}.Append(category.String()).Append(escapedFolders...)
+
+	return uniqueLoc{
+		pb:          pb,
+		prefixElems: 1,
+	}, nil
+}
+
+// TeamsChatsInfo describes a chat within teams chats.
+type TeamsChatsInfo struct {
+	ItemType   ItemType  `json:"itemType,omitempty"`
+	Modified   time.Time `json:"modified,omitempty"`
+	ParentPath string    `json:"parentPath,omitempty"`
+
+	Chat ChatInfo `json:"chat,omitempty"`
+}
+
+type ChatInfo struct {
+	CreatedAt          time.Time `json:"createdAt,omitempty"`
+	HasExternalMembers bool      `json:"hasExternalMemebers,omitempty"`
+	LastMessageAt      time.Time `json:"lastMessageAt,omitempty"`
+	LastMessagePreview string    `json:"preview,omitempty"`
+	Members            []string  `json:"members,omitempty"`
+	MessageCount       int       `json:"size,omitempty"`
```

Comment:
Should we keep field names & tag names identical? 
```suggestion
	MessageCount       int       `json:"messageCount,omitempty"`
```

Reply:
yes!  that's a bug, thanks for the catch.